exercises/practice/armstrong-numbers/.meta/config.json

```diff
@@ -3,13 +3,13 @@
   "authors": [],
   "files": {
     "solution": [
-      "src/main/kotlin/ArmstrongNumbers.kt"
+      "src/main/kotlin/ArmstrongNumber.kt"
     ],
     "test": [
-      "src/test/kotlin/ArmstrongNumbersTest.kt"
+      "src/test/kotlin/ArmstrongNumberTest.kt"
     ],
     "example": [
-      ".meta/src/main/kotlin/ArmstrongNumbers.kt"
+      ".meta/src/main/kotlin/ArmstrongNumber.kt"
```

Comment:
Their actual path is `.meta/src/reference/kotlin/*.kt` 🤔

Reply:
Ah, my mistake. I'll fix next week!

Reply:
Not a big deal, actually. I can fix this post-factum.